sourcegraph/about-replatform
# Add code search page

Closes #74 by adding the code search page and its components.

## To Test
- Check out `/code-search` and make sure that everything looks okay on desktop/mobile
